README.md

```diff
@@ -45,7 +44,23 @@ if [ -d "your/path/stuff/goes/here/pandamonium" ]; then
 fi
 ```
 
+### Install from TestPyPI
+
+You can install the latest development release of
+[`pandamonium` from TestPyPI][pandamonium_TestPyPI] into any Python virtual
+environment by running
+
+```
+python -m pip install --extra-index-url https://test.pypi.org/simple/ --pre pandamonium
+```
+
+> **Note:** This adds TestPyPI as [an additional package index to search][additional_package_index]
+when installing `pandamonium` specifically.
+PyPI will still be the default package index `pip` will attempt to install from
+for all dependencies.
+
 [pandamonium_TestPyPI]: https://test.pypi.org/project/pandamonium/
+[additional_package_index]: https://pip.pypa.io/en/stable/reference/pip_install/#cmdoption-extra-index-url
```

Comment:
I kind of think this block (and the one below it on a remove server) belongs lower down. Ideally it will only take someone the first 20 lines or so to get to the point where they see pretty colors.

Reply:
@dguest So maybe under a new section further down like `developing`? Do you have other thoughts?

Reply:
I guess the layout right now isn't terrible either, in that installation and running are separated.... Maybe I just empathize with all those people who don't like reading, and aren't very good at skimming, and who will, 90% of the time, get everything they need from 30 of the 180 lines here.

But your opinion is also welcome obviously.

Reply:
> empathize with all those people who don't like reading, and aren't very good at skimming, and who will, 90% of the time, get everything they need from 30 of the 180 lines here.

I think that's legitimate. This is maybe something that is more interesting to have in a `CONTRIBUTING` (c.f. Issue #31) and so if moved further down in the `README` could get migrated there later.

Reply:
Well some of it is just more complicated installation instructions, which I'd put under something like "Installation troubleshooting" or something right after "other tricks".

Reply:
Sorry, I saw this after I pushed. But this is now rebased off of the badges PR and also has a "Additional Technical Information" section added after the "Other tricks" section.